app.js

```diff
@@ -184,3 +184,16 @@ document.getElementById('button-see-mypost').addEventListener('click', () => {
 document.getElementById('modal-close').addEventListener('click', () => {
   document.getElementById('modal-container').style.display = 'none';
 });
+
+const mail = document.getElementById('mail');
+const form = document.getElementById('form');
+const word = document.getElementById('mistake');
+
+form.addEventListener('submit', (e) => {
+  if (mail.value === mail.value.toLowerCase()) {
+    word.textContent = '';
+  } else {
+    e.preventDefault();
+    word.textContent = 'The email must be in lowercase';
+  }
+});
```

Comment:
I make it optional but this is a requirement you should do it last milestones. Kindly fix it.
- [ ] I think you added some style while filling the form text size increase kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/190676592-d6970e68-68ce-426e-96e4-5ca7695e4d33.png)
![image](https://user-images.githubusercontent.com/38635158/190676833-049fbffc-f566-4977-9d5e-ef19b04f0d04.png)